### ISSUE TITLE
Configuration for the field name

### ISSUE THREAD

**Issue by kevmoo** [kevmoo/build_version#3]

**Comment by kevmoo** [comment]
This is blocked by https://github.com/dart-lang/build/issues/683

I think the best path forward would be something like https://github.com/kevmoo/build_version/issues/4

**Comment by lexaknyazev** [comment]
I don't think that this issue (configuring field name) is blocked by `buildExtensions`.

**Comment by kevmoo** [comment]
> I don't think that this issue (configuring field name) is blocked by `buildExtensions`.

Ah! You're right!
